src/ParallelAlgorithms/DiscontinuousGalerkin/InitializeDomain.hpp

```diff
@@ -61,12 +61,16 @@ namespace Actions {
  */
 template <size_t Dim>
 struct InitializeDomain {
-  using initialization_tags = tmpl::list<domain::Tags::InitialExtents<Dim>>;
+  using initialization_tags =
+      tmpl::list<domain::Tags::InitialExtents<Dim>,
+                 domain::Tags::InitialRefinementLevels<Dim>>;
 
-  template <typename DataBox, typename... InboxTags, typename Metavariables,
-            typename ActionList, typename ParallelComponent,
-            Requires<db::tag_is_retrievable_v<domain::Tags::InitialExtents<Dim>,
-                                              DataBox>> = nullptr>
+  template <
+      typename DataBox, typename... InboxTags, typename Metavariables,
+      typename ActionList, typename ParallelComponent,
+      Requires<tmpl::all<initialization_tags,
```

Comment:
Not for this PR, but if this is going to be a common pattern, should we make a metafunction, something like `db::databox_contains_v<DataBox,Tags>`?

Reply:
If it comes up again, let's do that.

In the long run, I hope we can implement #1971 and make all these checks obsolete, but adding a metafunction sounds easier.